_posts/en/newsletters/2022-05-25-newsletter.md

```diff
@@ -146,6 +146,38 @@ answers posted since our last update.*
 {% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
 {% assign bse = "https://bitcoin.stackexchange.com/a/" %}
 
+- [How much entropy is lost alphabetising your mnemonics?]({{bse}}113432)
+  HansBKK wonders how much entropy is lost if 12 or 24 word seed phrases were sorted
+  alphabetically. Pieter Wuille breaks down a series of metrics including number
+  of possibilities, entropy, and mean number of guesses to brute force across 12 and 24
+  words and also notes considerations for word repetition.
+
+- [Taproot signing with PSBT: How to determine signing method?]({{bse}}113489)
+  Guggero lists three ways of providing a valid [Schnorr signature][topic
+  schnorr signatures] in taproot: a keypath spend with [BIP86][] commitment, a
+  keypath spend with a commitment to the root of the script tree, and a
+  scriptpath spend. Andrew Chow confirms Guggero's outline of how each signing
+  method is indicated within a [PSBT][topic psbt].
+
+- [How would faster blocks cause mining centralization?]({{bse}}113505)
+  Murch focuses on why shorter block times would cause more frequent
+  reorganizations and benefit larger miners and how block propagation latency
+  plays a role.
```

Comment:
The double and was a bit strange. How do you like:

```suggestion
  Murch focuses on why shorter block times would cause more frequent
  reorganizations and how those would benefit larger miners in the context 
  of block propagation latency.
```